### PR TITLE
creating Model and Data schemas that reference each other

### PR DESCRIPTION
**Pull request by aconsilvio**
Setting up database structure

### CODE REVIEW

File: models/studentDataModel.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+var dataSchema = mongoose.Schema({
+	type: String,
+	date: Date,
+	data: Array
+});  
+​
+var studentSchema = mongoose.Schema({
+	id: Number,
+	name: String,
+	program: String,
```
Comment (nmohamed) [inline models/studentDataModel.js:15]: add
`archived: {type: Boolean, required: false}`
so it's true or doesn't exist (or just false, but would just make adding new students a little bit simpler)

File: models/studentDataModel.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+});
+​
+/*
+{
+	type: 'attendace'
+	date: 'Jul 01 2016'
+}
+*/
+​
+module.exports.attendace = mongoose.model('attendace', dataSchema);
```
Comment (davidabrahams) [inline models/studentDataModel.js]: I think convention is to [capitalize schema names](http://mongoosejs.com/docs/populate.html)

File: models/studentDataModel.js (hunk 1 above)
Comment (davidabrahams) [inline models/studentDataModel.js:15]: we don't have to do the required part

File: models/studentDataModel.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+	data: [{type: Schema.ObjectId, required: false, ref: 'data'}],
+	archived: {type: Boolean, required: false}
+});
+
+var userSchema = mongoose.Schema({
+	username: String,
+	password: String
+});
+​
+/*
```
Comment (davidabrahams) [inline models/studentDataModel.js]: remove this?

**Comment by davidabrahams** [comment]
mergy merge

**Pull request merged**